README.md

```diff
@@ -72,159 +72,159 @@ Identity Providers are responsible for validating the identity of Swift Citizens
+* Choose the inflation rate in [Stage Two](#stage-two).
+* Vote on the [salary for Delegated Nodes](#delegated-node-salary).
+* Vote on the [Salary Multiplier for Identity Providers](#identity-provider-salary).
+* Vote to approve or disapprove [funding proposals](#funding-proposals).
+
+To perform these abilities, a proposal must be submitted to the blockchain by one Delegated Node. All other Delegated Nodes on the network will then vote on the proposal. Any proposal that receives 50% + 1 votes within 24 hours of submission will be executed.
 
 ## Transaction Capabilities
 
 ### Transactions Per Second
-Due to the controlled nature of the Delegated Node network, nodes can work to forge blocks extremely fast with a targeted time of 3 seconds per block. Other algorithms that have implemented proof of stake algorithms such as EOS which has the same functionality of having selected speaker nodes that forge blocks\[1] Have proven that it's possible to scale such a system to 50,000 Transactions Per Section\[2] For comparison Visa is able to scale up to 24,000 TPS \[3]
+Due to the controlled nature of the Delegated Node network, nodes can work to forge blocks extremely fast with a targeted time of three seconds per block. Other algorithms that have implemented proof-of-stake algorithms (such as EOS--which has the same functionality of having selected speaker nodes that forge blocks<sup>[[2]](#2)</sup>) have proven that it's possible to scale such a system to 50,000 transactions per second<sup>[[3]](#3)</sup>. For comparison, Visa is able to scale to 24,000 TPS <sup>[[4]](#4)</sup>.
 
 ### Low Transaction Fees
-Transactions have no inherent cost to send on SwiftDemand, however Identity Providers do have the ability to set transaction fees allowing them to have funds to resolve transaction disputes. The Swift Protocol reserves 80% of each block to be used exclusively for Identity Providers proportional to the amount of Swift Citizens they have verified with a minimum of 10. It is therefore the responsibility of Identity Providers to ensure spam transactions are not added to the chain. The remaining 20% of each block will be open for transactions with a bidding system in a similar manner to how bitcoin functions \[4]
+Transactions have no inherent cost on SwiftDemand, however [Identity Providers](#identity-providers) have the ability to set transaction fees--allowing them to have funds to resolve transaction disputes. The Swift Protocol reserves 80% of each block to be used exclusively by Identity Providers. The space is further partitioned by Identity Provider--proportional to the number of Swift Citizens each has verified, with a minimum requirement for inclusion of ten Swift Citizens. It's therefore the responsibility of Identity Providers to ensure that spam transactions are not added to the chain. The remaining 20% of each block will be open for transactions with a bidding system--in a similar manner to how bitcoin functions <sup>[[5]](#5)</sup>.
```

Comment:
The amount of transactions that an Identity Provider is able to have included on a single block can't go below 10 to allow new Identity Providers to add new Swift Citizens and make initial transactions.

Reply:
Ok, I understand what you're trying to say.  Change "amount" to "number" and "10" to "ten."